DESCRIPTION

```diff
@@ -28,6 +28,5 @@ Suggests:
     covr,
```

Comment:
Hi @tdhock ,

Removing fpop from SUGGESTS works and the build passes on Travis CI even though I get one note.

Reply:
ok, @guillemr is working on getting fpop on CRAN though